www/index.html

```diff
@@ -38,23 +20,154 @@
+
+    <div class="container">
+        <div class="page-header">
+            <h1>Fln Routes</h1>
+            <p class="lead">Simple Florianópolis' public transportation route searcher.</p>
+        </div>
+
+        <div class="list-view">
+            <form action="#" class="form-inline route-search-form">
+                <label class="sr-only" for="location">Location</label>
+                <input type="text" id="location" name="location" class="form-control" placeholder="Location">
+                <button type="submit" class="btn btn-primary">Go! <i class="fa fa-spinner fa-pulse fa-fw loading-indicator"></i></button>
+            </form>
+
+            <div class="routes-panel"></div>
+        </div>
+
+        <div class="detail-view hide">
+            <button type="button" class="btn pull-right back-trigger">Voltar</button>
+            <div class="container-fluid">
+                <h4 class="route-name-label"></h4>
+                <i class='fa fa-spinner fa-pulse fa-2x fa-fw loading-indicator loading-indicator'></i>
+                <div class="row">
+                    <div class="col-md-6 stops-panel"></div>
+                    <div class="col-md-6 departures-panel"></div>
+                </div>
+            </div>
+            <button type="button" class="btn pull-right back-trigger">Voltar</button>
         </div>
     </div>
-    <script type="text/javascript" src="cordova.js"></script>
-    <script type="text/javascript" src="js/index.js"></script>
-    <script type="text/javascript">
-        app.initialize();
+
+    <script id="no-input-error-tmpl" type="x-tmpl-mustache">
+        <div class="alert alert-danger alert-dismissible fade in" role="alert">
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+            </button>
+            <strong>Oops!</strong> É preciso definir um nome de localidade.
+        </div>
     </script>
+
+    <script id="stops-tmpl" type="x-tmpl-mustache">
+        {{ ^stops }}
+            <div class="alert alert-warning" role="alert">
+                <strong>Oops!</strong> Nenhum registro encontrado.
+            </div>
+        {{ /stops }}
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Paradas</h3>
+            </div>
+            <div class="panel-body">
+                <ol>
+                    {{ #stops }}
+                        <li>{{ name }}</li>
+                    {{ /stops }}
+                </ol>
+            </div>
+        </div>
+    </script>
+
+    <script id="departures-tmpl" type="x-tmpl-mustache" >
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Dia de Semana</h3>
+            </div>
+            <div class="panel-body">
+                <ul class="list-inline unstyled">
+                    {{ #WEEKDAY }}
```

Comment:
I decided to keep it upper case because that is the way the value is returned from API and I thought it might be easier to "map" the template with API's result. 

But, indeed, it might be a good idea to use lower case to keep code consistency.